src/main/kotlin/io/kuzzle/sdk/coreClasses/exceptions/MissingURLParamException.kt

```diff
@@ -0,0 +1,8 @@
+package io.kuzzle.sdk.coreClasses.exceptions
+
+/**
+ * Thrown when attempting to interact with the network while not connected.
+ */
+class MissingURLParamException : KuzzleException {
+    constructor(templateParam: String, baseURL: String) : super("Missing URL Param $templateParam in $baseURL", 0)
+}

```

Comment:
`class MissingURLParamException(templateParam: String, baseURL: String) : KuzzleException("Missing URL Param $templateParam in $baseURL", 0)`